src/Aspire.ProjectTemplates/templates/aspire-starter/9.1/Aspire-StarterApplication.1.ApiService/Aspire-StarterApplication.1.ApiService.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="!!REPLACE_WITH_ASPNETCORE_9_VERSION!!" Condition=" '$(Framework)' == 'net9.0' " />
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="9.0.2" Condition=" '$(Framework)' == 'net9.0' " />
```

Comment:
Is that correct?

Reply:
Yep: https://www.nuget.org/packages/Microsoft.AspNetCore.OpenApi/9.0.2

Reply:
TBC, we only keep the prior Aspire version templates in here to not rug-pull users when they update to the latest version of the templates package, but we move them to static, unupdated mode at that point.